### PR TITLE
#157340050 Integrate Circle CI

### PR DESCRIPTION
**Pull request by EmaEvidence**
## What does this PR do?
- Integrates Circle CI

## Description of Task to be completed?
- Integrate Circle CI

## How should this be manually tested?
- Commit a change.
- Push the change
- Raise a PR for the change

## What are the relevant pivotal tracker stories?
[#157340050](https://www.pivotaltracker.com/story/show/157340050)
## Any background context you want to add?
- Nil
## Important notes
- Nil
## Packages installed
- Nil
## Deployment note
- Nil
## Related PRs branch | PR (branch_name) | (pr_link)
- Nil
## Todos
- Nil
## Screenshots
- Nil

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          name: run tests
+          command: yarn test
+
+      # - run:
+      #     name: run coverage
+      #     command: sudo yarn coverage
+
+      # - run:
+      #     name: report coverage
+      #     command: sudo yarn coveralls
```
Comment (HawiCaesar) [inline .circleci/config.yml]: You can add a line at the end here and git will ignore that extra line.

**Comment by HawiCaesar** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
+[![CircleCI](https://circleci.com/gh/AndelaOSP/art-dashboard/tree/ch-integrate-circle-ci-157340050.svg?style=svg)](https://circleci.com/gh/AndelaOSP/art-dashboard/tree/ch-integrate-circle-ci-157340050)
+
```
Comment (wanjiku-carol) [inline README.md]: The badge integrates to the branch specifically. I think it should be directed to the default branch.

File: .circleci/config.yml (hunk 1 above)
Comment (EmaEvidence) [reply .circleci/config.yml]: Fixed, Thanks.

**Comment by EmaEvidence** [comment]
I wanted to configure Circle CI to run coverage but it is not available the build is failing that was why I commented it out. @theterminalguy

**Pull request merged**